application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -34,6 +36,11 @@
 public class TeXCommand extends SlashCommandAdapter {
 
     private static final String LATEX_OPTION = "latex";
+    // everything between 2 $'s
```

Comment:
I think an example would help as well.
```suggestion
    // Matches regions between two dollars, like '$foo$'.
```